### DISCUSSION TITLE
unable to import @v3 of node-fetch

### DISCUSSION THREAD

**Discussion by VivekLande** [node-fetch/node-fetch#1584] [Q&A]
Hi Team, 
I have tried all the solution provided at [Loading node-fetch@3 in CJS and ESM](https://github.com/node-fetch/node-fetch/issues/1279) but nothing is working and getting below error   'SyntaxError: Cannot use import statement outside a module'
Please help.
![image](https://user-images.githubusercontent.com/16791810/173514951-5c7b387e-2ebb-44d7-8914-41a98c528f77.png)

**Comment by jimmywarting** [comment]
if you have switched to using ESM over commonjs then you should probably also have `{ "type": "module" }` in your package.json

**Reply by VivekLande** [reply]
Hi @jimmywarting,

I have updated my package.json with { "type": "module" } and tsconfig.json too with ["module": "ES2020"] as you suggest but it couldn't help.

**Reply by VivekLande** [reply]
Looking to get work, can you please help?

**Reply by jimmywarting** [reply]
what kind of error do you get when using adding type module into package.json? 

©sindresorhus have a guide to convert to esm. https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

**Reply by VivekLande** [reply]
Hi @jimmywarting, please see below error snapshot even on all configuration
![image](https://user-images.githubusercontent.com/16791810/173527225-c6cf147f-92bd-4843-940d-0bdeb0d21601.png)

**Reply by jimmywarting** [reply]
the error seems to come from jest.
https://jestjs.io/docs/ecmascript-modules

fyi, nodejs v18+ have fetch built in now.

**Reply by VivekLande** [reply]
Thanks @jimmywarting, let me see how I can handle it with jest.
